notify/grafana_alertmanager.go

```diff
@@ -139,7 +139,6 @@ var NewIntegration = notify.NewIntegration
 type InhibitRule = config.InhibitRule
 type MuteTimeInterval = config.MuteTimeInterval
 type TimeInterval = timeinterval.TimeInterval
-type Route = config.Route
```

Comment:
I wonder if it is used in Grafana

Reply:
I didn't remove it, just moved to the new [routes.go](https://github.com/grafana/alerting/pull/110/files#diff-780c314ab98f5ca2f76594c489af92e51fe9545abb849ad05b27a910e4946e7aR5)